salsa-2022-tests/tests/compile-fail/specify-does-not-work-if-the-key-is-a-salsa-input.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0277]: the trait bound `MyInput: TrackedStructInDb<dyn Db>` is not satisfied
+   --> tests/compile-fail/specify-does-not-work-if-the-key-is-a-salsa-input.rs:21:28
```

Comment:
It says "test/compile-fail/specify-does-not-work-if-the-key-is-a-salsa-input.rs:20:1" on the nightly version, the difference is in the span